strings/Java String Reverse/Solution.java

```diff
@@ -0,0 +1,21 @@
+import java.lang.String;
+import java.lang.StringBuilder;
+import java.util.Scanner;
+
+public class Solution {
+  
+  private static String checkIfPolindrom(String inputString) {
+    StringBuilder reversedString = new StringBuilder(inputString);
```

Comment:
Good choice :) Just one suggestion, since you're actually using reversed string you don't need to store StringBuilder as it is a bit misleading. So I'd put

`String reversed = new StringBuilder(input).reverse().toString()`

Also no need for type in names, I mean drop `String` part in variables since it's known from it's type anyway.